core/src/test/scala/com/broadinstitute/dsp/Generators.scala

```diff
@@ -11,7 +11,7 @@ object Generators {
     cloudService <- genCloudService
     project <- genGoogleProject
     runtimeName <- Gen.uuid.map(_.toString)
-    status <- Gen.oneOf("Running", "Creating", "Deleted", "Error")
+    status <- Gen.oneOf("RUNNING", "CREATING", "DELETED", "ERROR")
```

Comment:
the origina generator is correct...`status` for `Runtime` is not all upper case

Reply:
Ah I think I confused `Runtime` which models the Leo DB query result with `Instance` (from Google library) status which _is_ all upper case.